packages/core/cmake/soss_mix_generator.cmake

```diff
@@ -681,7 +681,7 @@ function(_soss_mix_find_package_info)
     # list(GET ...). Now we convert each of those colon-separated lists into
     # semicolon-separated lists so that cmake can recognize them correctly as
     # lists.
-    string(REPLACE ":" ";" ${output_var} "${${output_var}}")
+    string(REPLACE "#" ";" ${output_var} "${${output_var}}")
```

Comment:
I've changed `colon` for `hash` in the comments

Reply:
Just out of curiosity, was there a specific motivation for this? Is there an issue where the colon can be mistaken for the disk designator, like `C:\`?

I'm a bit concerned that the hash sign is considered a valid character for both Linux and Windows. But technically the semi-colon is a valid filename character too, so :man_shrugging: 

I guess if people decide to put hashes in their filenames, they've taken their fate into their own hands.

Reply:
Yes, is exactly that. If we use colon we have troubles with C:\ resulting in invalid paths. We put hashes because is not a common character for a path but you are right, this may be a problem. Actually, we can put here anything rather than colon because global paths in windows have colons.

Reply:
Yeah, let's just stick with hashes, then for lack of a better alternative.